### PR TITLE
Update live with current master

### PR DESCRIPTION
**Pull request by natke**
Prevew link: https://review.docs.microsoft.com/en-us/quantum/index?view=qsharp-preview&branch=pr-en-us-555

### CODE REVIEW

File: articles/language/type-model.md [hunk 1]
```diff
@@ -472,3 +472,70 @@ a valid invocation of `Controlled Rz` (note the parentheses around `0.1, target`
 
 As another example, `CNOT(control, target)` can be implemented as `Controlled X([control], target)`. 
 If a target should be controlled by 2 control qubits (CCNOT), we can use `Controlled X([control1, control2], target)` statement.
+
+### Examples
+
+This example of a Q# operation comes from the [Measurement](https://github.com/Microsoft/Quantum/tree/master/Samples/Measurement) sample. Within operations, we can allocate qubits and use quantum operations on those qubits such as `H` and `X`:
```
Comment (cgranade) [inline articles/language/type-model.md]: Looks like this link got missed earlier, my apologies for that.
```suggestion
This example of a Q# operation comes from the [Measurement](https://github.com/microsoft/Quantum/tree/master/samples/getting-started/measurement) sample. Within operations, we can allocate qubits and use quantum operations on those qubits such as `H` and `X`:
```

File: articles/language/type-model.md [hunk 2]
```diff
@@ -472,3 +472,70 @@ a valid invocation of `Controlled Rz` (note the parentheses around `0.1, target`
...
+            if (result == One) {    // Reset the qubit so that it can be released
+                X(qubit);
+            }
+
+            return result;
+        }
+ }
+```
+
+This example of a function comes from the [PhaseEstimation](https://github.com/Microsoft/Quantum/tree/master/Samples/PhaseEstimation) sample. It contains purely classical code. You can see that, unlike the example above, no qubits are allocated, and no quantum operations are used.
```
Comment (cgranade) [inline articles/language/type-model.md]:
```suggestion
This example of a function comes from the [PhaseEstimation](https://github.com/microsoft/Quantum/tree/master/samples/characterization/phase-estimation) sample. It contains purely classical code. You can see that, unlike the example above, no qubits are allocated, and no quantum operations are used.
```

File: articles/language/type-model.md [hunk 3]
```diff
@@ -472,3 +472,70 @@ a valid invocation of `Controlled Rz` (note the parentheses around `0.1, target`
...
+    mutable product = new Double[Length(left)];
+
+    for (idxElement in IndexRange(left)) {
+        set product w/= idxElement <- left[idxElement] * right[idxElement];
+    }
+    return product;
+}
+```
+
+It is also possible for a function to be passed qubits for processing, as in this example from the [ReversibleLogicSynthesis](https://github.com/Microsoft/Quantum/tree/master/Samples/ReversibleLogicSynthesis) sample. Qubits are passed to the function and used for processing, although at no point are the qubit states themselves modified.
```
Comment (cgranade) [inline articles/language/type-model.md]:
```suggestion
It is also possible for a function to be passed qubits for processing, as in this example from the [ReversibleLogicSynthesis](https://github.com/microsoft/Quantum/tree/master/samples/algorithms/reversible-logic-synthesis) sample. Qubits are passed to the function and used for processing, although at no point are the qubit states themselves modified.
```

**Comment by natke** [comment]
Thanks @cgranade,  I'll open a separate PR to fix those issues. I like to keep the master live merge a simple collection of the PRs we're merged to master. It's easier to track what is what that way.

**Comment by cgranade** [comment]
> Thanks @cgranade, I'll open a separate PR to fix those issues. I like to keep the master live merge a simple collection of the PRs we're merged to master. It's easier to track what is what that way.

Makes perfect sense, my apologies again for not noticing those broken links in earlier PRs.

**Pull request merged**